libsolidity/formal/SymbolicState.cpp

```diff
@@ -94,12 +94,12 @@ smtutil::Expression SymbolicState::balance() const
 
 smtutil::Expression SymbolicState::balance(smtutil::Expression _address) const
 {
-	return smtutil::Expression::select(balances(), move(_address));
+	return smtutil::Expression::select(balances(), std::move(_address));
 }
 
 smtutil::Expression SymbolicState::blockhash(smtutil::Expression _blockNumber) const
 {
-	return smtutil::Expression::select(m_tx.member("blockhash"), move(_blockNumber));
+	return smtutil::Expression::select(m_tx.member("blockhash"), std::move(_blockNumber));
```

Comment:
```suggestion
	return smtutil::Expression::select(
	     m_tx.member("blockhash"), 
	     std::move(_blockNumber)
	);
```

Reply:
I'd argue that we shouldn't change the formatting (at least not in this PR), especially if our `check_style` script doesn't complain. I'd much rather prefer to wait until `clang-format` starts supporting everything we want (which should be soon), and then we can just use that to format our code. In addition, it should significantly simplify our `check_style` script, as it should simply do a comparison between current file and `clang-format-X current-file`, and if there's a mismatch, fail the step.